pkg/utils/validation/validation_suite_test.go

```diff
@@ -1,13 +1,13 @@
-package table_test
+package validation
 
 import (
+	"testing"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-
-	"testing"
 )
 
-func TestTable(t *testing.T) {
+func TestFeatureGate(t *testing.T) {
```

Comment:
Something is strange here. It looks like you moved a file from vendor to here.
Also, please use a better name for this function, best practice IIRC is Test<PackageName>.

Reply:
No problem! I'll get on suggestions, agree across the board and just needed some nudges on what maintainers would prefer here and there. I'll also get that git history confusion cleaned up, not sure what caused that in a new file addition :\